File: java/program1.java
import java.util.*;

float CalculateArea(float fvalue)
{

float fAns=0.0f;
float PI=3.14f;
fAns=PI*fvalue*fvalue;
}
class main
{
public static void main(String args[])
{
    float fRadius = 0.0f;
    float fArea =0.0f;
    Scanner sc=new Scanner(System.in);
System.out.println("Enter radius number:");
    float fRadius=sc.nextFloat();
    fArea = CalculateArea(fRadius);

    System.out.println("Area of circle is :" +fArea);
}
}

File: c++/program1.c
#include<stdio.h>

float CalculateArea(float fvalue)
{

auto float fAns=0.0f;
auto float PI=3.14f;
fAns=PI*fvalue*fvalue;
return fAns;
}
int main()
{
    auto float fRadius = 0.0f;
    auto float fArea =0.0f;

    printf ("Enter radius number:");
    scanf("%f",&fRadius);
    fArea = CalculateArea(fRadius);

    printf("Area of circle is :%f\n",fArea);
    return 0;
}
